src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4853methodsandconstructorsannotations/InputMethodsAndConstructorsAnnotations.java

```diff
@@ -0,0 +1,75 @@
+package com.google.checkstyle.test.chapter4formatting.rule4853methodsandconstructorsannotations;
+
+public class InputMethodsAndConstructorsAnnotations {
```

Comment:
> Exception: A single parameterless annotation may instead appear together with the first line of the signature

Two cases are missed:
1. Multiple annotations on the same line with the method signature. We expect a warning here.
2. A **single parameterised** annotation on the same line with the method signature. We expect a warning here.

Reply:
Done. 

https://github.com/checkstyle/checkstyle/blob/859ba58c92f71d16f922479a2b228222b2cd5b0b/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4853methodsandconstructorsannotations/InputMethodsAndConstructorsAnnotations.java#L41-L43

https://github.com/checkstyle/checkstyle/blob/859ba58c92f71d16f922479a2b228222b2cd5b0b/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4853methodsandconstructorsannotations/InputMethodsAndConstructorsAnnotations.java#L78-L80

I have considered both, method and constructor for this as it was not specifically mentioned which signature type they are talking about.